### PR TITLE
Move browser related code from main to Browser class

### PR DESCRIPTION
**Pull request by rainemak**
Moving code around. I was doing this in different context some days back. Didn't want this one to grow bigger than this.

Should be rather easy to review with gitk or similar that shows file move diff correctly.

### CODE REVIEW

File: src/browser/browser_p.h [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+****************************************************************************/
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BROWSER_PRIVATE_H
+#define BROWSER_PRIVATE_H
+
+#include "closeeventfilter.h"
```
Comment (siteshwar) [inline src/browser/browser_p.h]: Just declaring ```class CloseEventFilter``` should be enough here.

**Comment by siteshwar** [comment]
LGTM

File: src/browser/browser_p.h (hunk 1 above)
Comment (rainemak) [inline src/browser/browser_p.h]: Yes, on the other hand this is private class. So, doesn't matter that much.

Comment (rainemak) [inline src/browser/browser_p.h]: Updated.

**Pull request merged**